safenode/src/client/api.rs

```diff
@@ -210,7 +211,12 @@ impl Client {
             match select_all(list_of_futures).await {
                 (Ok(res), _, remaining_futures) => {
                     let res = res.map_err(Error::Network);
-                    info!("Got response for the req: {req:?}, res: {res:?}");
+                    let res_string = match &res {
```

Comment:
Bit ugly, but the value must be extracted from the result it seems, for it to not override the implemented display, which is meant to cut down on very large and verbose details.